changelog.d/20241008_214456_30907815+rjmello_compute_client_function_methods.rst

```diff
@@ -0,0 +1,6 @@
+Added
+~~~~~
+
+- Added a ``ComputeClient.register_function()`` method. (:pr:`NUMBER`)
+
+- Added a ``ComputeClient.delete_function()`` method. (:pr:`NUMBER`)
```

Comment:
I don't feel strongly about phrasing, but I think these should be combined:

```suggestion
- Add ``ComputeClient.register_function()`` and ``ComputeClient.delete_function()``. (:pr:`NUMBER`)
```

Also, I think another bullet point is needed for the addition of helper classes.